docker/Dockerfile.istiod

```diff
@@ -11,7 +11,7 @@ USER 1337:1337
 
 # The following section is used as base image if BASE_DISTRIBUTION=distroless
 # hadolint ignore=DL3007
-FROM gcr.io/distroless/static:nonroot as distroless
+FROM discolix/static:nonroot as distroless
```

Comment:
was this change intentional? I don't see discolix defined on Docker Hub or grc.io

Reply:
Nevermind. Found it on Dockerhub. 
https://hub.docker.com/r/discolix/static

Reply:
I would prefer to see distroless divided by architecture, or even better for gcr.io/distroless/static to support arm targets. The discolix image is nearly 2 months old as of this review.